content/en/data_jobs/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Data Jobs Monitoring
+kind: documentation
+further_reading:
+    - link: '/data_streams'
+      tag: 'Documentation'
+      text: 'Data Streams Monitoring'
+
+---
+
+{{< callout url="https://forms.gle/PZUoEgtBsH6qM62MA" >}}
+Data Jobs Monitoring is in private beta. Fill out this form to join the wait list.
+{{< /callout >}} 
+
+{{< img src="data_jobs/overview.png" alt="Datadog Data Jobs Monitoring" style="width:100%;" >}}
+
+Data Jobs Monitoring provides visibility into the performance and reliability of data processing jobs, including Apache Spark and Databricks jobs, along with the underlying infrastructure. Data Jobs Monitoring enables you to:
+
+- Track data processing jobs across your accounts and workspaces, and see which take up the most compute resources or have inefficiencies.
+- Receive an alert when a job fails—or when a job that should be complete is still running.
+- Analyze job execution details and stack traces.
+- Correlate infrastructure metrics, Spark metrics from the Spark UI, logs, and cluster configuration.
+- Compare multiple runs to facilitate troubleshooting, and to optimize provisioning and configuration during deployment.
+
+## Setup
+
+Data Jobs Monitoring is supported for Amazon EMR, Databricks, and Spark on Kubernetes.
+
+To get started, select your platform and follow the installation instructions:
+
+{{< partial name="data_jobs/setup-platforms.html" >}}
+
+<br/>
+
+## Explore Data Jobs Monitoring
+
+### Take inventory of your data processing jobs
+
+View all jobs across cloud accounts and workspaces. See CPU allocation, cost estimation, and other metrics to identify opportunities for optimization.
```

Comment:
```suggestion
View all jobs across cloud accounts and workspaces. Identify failing jobs to take action on, or those with high idle CPU that are using a lot of compute that should be optimized.
```

Reply:
View all jobs across cloud accounts and workspaces. Identify failing jobs to take action on, or find jobs with high idle CPU that are using a lot of compute and should be optimized.